packages/mobile/src/send/Send.tsx

```diff
@@ -90,7 +93,8 @@ const mapStateToProps = (state: RootState): StateProps => ({
   verificationPossible: verificationPossibleSelector(state),
   devModeActive: state.account.devModeActive,
   recentRecipients: state.send.recentRecipients,
-  allRecipients: phoneRecipientCacheSelector(state),
+  phoneRecipients: phoneRecipientCacheSelector(state),
+  valoraRecipients: valoraRecipientCacheSelector(state),
```

Comment:
If one of our contacts has a Valora account, will it show up twice? Once in the Valora section and another time in the phone contacts section?

Reply:
yes that's true!